Authenticator/Source/TokenListViewModel.swift

```diff
@@ -28,4 +28,11 @@ import Foundation
 struct TokenListViewModel {
     let rowModels: [TokenRowModel]
     let ringProgress: Double?
+    let totalTokens: Int
+    var isFiltering: Bool {
+        return totalTokens > rowModels.count
```

Comment:
Could this have unexpected side effects if the filter applies to (and shows) every available token? (That is, if the filter is active, but all tokens pass the filter so `rowModels.count` still equals `totalTokens`.)